extensions/db/src/main/java/org/mule/extension/db/internal/domain/connection/mysql/MySqlDbUtils.java

```diff
@@ -37,25 +37,29 @@ private static String buildUrlFromAttributes(String urlPrefix, String host, Inte
       buf.append(port);
     }
     buf.append("/");
-    buf.append(database);
+    if (database != null) {
```

Comment:
In any DB connection the specification of the database name or schema is optional, due that you can specify it later in a query